modules/quoting.js

```diff
@@ -146,8 +146,8 @@ function convertForwardedToBlockquote(aDoc) {
       let txt = child.textContent;
       let m = txt.match(re);
       if (child.nodeType == child.TEXT_NODE
-          && txt.includes("-----BEGIN PGP")
-          && txt.includes("----END PGP")
+          && !txt.includes("-----BEGIN PGP")
```

Comment:
What is this change for?

Reply:
https://github.com/protz/thunderbird-conversations/commit/100641dbd663a3cebab55e279cc47e4078565821#diff-e54960bec9e47f840349eca45e3dcd94L171
Your commit changed that. It breaks inline pgp decription.
Not including is right.

Reply:
Thank you for spotting that. Looks like I must have changed it manually and got it wrong :-(

I'll try and find time to look at the patch one more time in the next day or so.